src/test_l1_plat_power.c

```diff
@@ -612,7 +614,7 @@ void test_l1_plat_power_positive_PLAT_API_GetWakeupSrc (void)
 {
     gTestID = 11;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
-	pmStatus_t ret;
+    pmStatus_t ret;
     bool enable1, enable2;
     bool lan_support = 0;
 
```

Comment:
check indentation of for and if loops.. check whole file

Reply:
Done

Reply:
replace tab space with space.. whole file

Reply:
Done